autogen/main.tf.tmpl

```diff
@@ -31,25 +31,27 @@ locals {
 
 ### IPv4 block ###
 resource "google_compute_global_forwarding_rule" "http" {
-  provider   = google-beta
-  project    = var.project
-  count      = local.create_http_forward ? 1 : 0
-  name       = var.name
-  target     = google_compute_target_http_proxy.default[0].self_link
-  ip_address = local.address
-  port_range = "80"
-  labels     = var.labels
+  provider              = google-beta
+  project               = var.project
```

Comment:
I had to use the variable name "project" despite "project_id" being preferred, as that was what was already here, and changing it would break backward compatibility